src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -2,25 +2,82 @@
 
```

Comment:
1. **Error Handling in `divide` Method**:
   - The `divide` method throws an `IllegalArgumentException` when dividing by zero, which is good practice. You may also consider using `ArithmeticException` instead, as it is more specific to mathematical errors.
  
2. **Input Validation**:
   - Consider adding input validation for all methods to handle edge cases or prevent unexpected behavior. For example, checking for null inputs or extreme values that may cause overflow or unexpected results.